modules/Cockpit/assets/components/field-layout.tag

```diff
@@ -229,6 +229,9 @@
             this.components = App.$.extend(true, this.components, window.CP_LAYOUT_COMPONENTS);
         }
 
+        if(opts.parentComponent && opts.parentComponent.options) {
+            opts = App.$.extend(true, opts, opts.parentComponent.options);
```

Comment:
shouldn't  it be the other way around  `opts = App.$.extend(true, opts.parentComponent.options, opts);`? that way the component options  can override the parent component options.

Reply:
Yes it should, well spotted!